### PR TITLE
Add Changes to Files

### PR DESCRIPTION
**Pull request by Donatos-Andrea**
edit javascript file - f436422b92e04c3ffc6d7971c1a9f48fdaac621f. 
edit the button type in html -  eeb19088a8f9e555cc3c184fc4a86c165b126283.
edit html file - 70a2d9b235ca48ab8bebf6ebe014fb2577598ca9.
edit js file - 39bf6073db1f9791b802bf13481d4662a21dcd12.

### CODE REVIEW

**Comment by Monroy-JeffCarl** [comment]
Hello, Andrea! I am Jeff and I am tasked to review your codes. Thank you!

File: donatos_andrea/index.js [hunk 1]
```diff
@@ -1,11 +1,82 @@
...
-        comment_button.disabled = false;
-    } else {
-        comment_button.disabled = true;
-    }
-} 
+	 if (memberName.value && commentArea.value) {
+         commentButton.disabled = false;
+     } else {
+         commentButton.disabled = true;
+}
```
Comment (Monroy-JeffCarl) [inline donatos_andrea/index.js]: Hi! Please add indentation to this closing brace in the code so that it aligns properly.

**Comment by Monroy-JeffCarl** [comment]
Also, make sure that the newly added comments are sorted correctly alongside hard-coded ones.

**Comment by Donatos-Andrea** [comment]
> Also, make sure that the newly added comments are sorted correctly alongside hard-coded ones.

hello po, hindi ko po masyado magets mga need ko mabago sana po palink ng mismong need ng changes. thank youuu

**Comment by Monroy-JeffCarl** [comment]
Hello! Apologies sa confusion. Sa end ko, ganito yung behavior ng comment section. And based sa step no. 4 sa exercise 13, the comments should be sortable by date. Thank you!
Screenshot 2024-05-15 234448



https://github.com/PUP-BSIT/exercise-13-tuf_developers/assets/146610699/5f6e6861-0105-4cbe-8676-0433fc8cebd6

**Comment by Monroy-JeffCarl** [comment]
But overall, your code looks good and the UI is impressive! Thank you again, Andrea!

**Review by Monroy-JeffCarl — APPROVED** [review]

**Comment by Donatos-Andrea** [comment]
> But overall, your code looks good and the UI is impressive! Thank you again, Andrea!

```fix date sorting bug``` - 686843ea98dd826b612d91d655e295cbe8310ee0

thank you po

**Pull request merged**